### PR TITLE
Added danish translation

### PR DESCRIPTION
**Pull request by jkiddo**

### CODE REVIEW

**Review by ieju — COMMENTED** [review]
Tiyt

File: lang/da/index.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
+---
+titel: Semantisk versionsstyring 2.0.0
+sprog: da
+forfatter: Jens Kristian Villadsen
```
Comment (JohnTitor) [inline lang/da/index.md]: Please keep them in English, these are metadata.
```suggestion
title: Semantisk versionsstyring 2.0.0
language: da
author: Jens Kristian Villadsen
```

**Comment by jkiddo** [comment]
@JohnTitor looks good

**Comment by JohnTitor** [comment]
Could you also fix the lint failure? I recently updated deps versions so rebasing onto the latest gh-pages branch may be required.

**Comment by jkiddo** [comment]
@JohnTitor should be good to go now I think

File: lang/da/spec/v2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
+---
+titel: Semantisk versionsstyring 2.0.0
+sprog: da
+forfatter: Jens Kristian Villadsen
```
Comment (JohnTitor) [inline lang/da/spec/v2.0.0.md]:
```suggestion
title: Semantisk versionsstyring 2.0.0
language: da
author: Jens Kristian Villadsen
```

File: lang/da/spec/v2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+         identifikatorer.
+
+      1. Et større sæt præ-udgivelsesfelter har en højere prioritet end en
+         mindre sæt, hvis alle de foregående identifikatorer er ens.
+
+      Eksempel: 1.0.0-alfa < 1.0.0-alfa.1 < 1.0.0-alfa.beta < 1.0.0-beta <
+      1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+
+
```
Comment (JohnTitor) [inline lang/da/spec/v2.0.0.md]:
```suggestion

```

File: lang/da/spec/v2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+Preston-Werner](https://tom.preston-werner.com), opfinder af Gravatar og
+medstifter af GitHub.
+
+Hvis du gerne vil give feedback, bedes du [åbn et issue på
+GitHub](https://github.com/semver/semver/issues).
+
+Licens
+-------
+
+[Creative Commons ― CC BY 3.0](https://creativecommons.org/licenses/by/3.0/)
```
Comment (JohnTitor) [inline lang/da/spec/v2.0.0.md]:
```suggestion
[Creative Commons ― CC BY 3.0](https://creativecommons.org/licenses/by/3.0/)

```

File: lang/da/spec/v2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+
+      1. Numeriske identifikatorer har altid lavere forrang end ikke-numeriske
+         identifikatorer.
+
+      1. Et større sæt præ-udgivelsesfelter har en højere prioritet end en
+         mindre sæt, hvis alle de foregående identifikatorer er ens.
+
+      Eksempel: 1.0.0-alfa < 1.0.0-alfa.1 < 1.0.0-alfa.beta < 1.0.0-beta <
+      1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+Hvorfor bruge semantisk versionering?
```
Comment (JohnTitor) [inline lang/da/spec/v2.0.0.md]:
```suggestion
      1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.

Hvorfor bruge semantisk versionering?
```

**Review by JohnTitor — APPROVED** [review]
Thanks for translating!

**Comment by jkiddo** [comment]
@JohnTitor crap - I forgot to add it to https://github.com/semver/semver.org/blob/gh-pages/_config.yml

**Comment by JohnTitor** [comment]
> I forgot to add it to [https://github.com/semver/semver.org/blob/gh-pages/_config.yml](https://github.com/semver/semver.org/blob/gh-pages/_config.yml?rgh-link-date=2022-07-23T12%3A05%3A06Z)

Oh well, could you open another PR? I'm not sure what's "key: value" correct (`da: Dansk`?)

**Pull request merged**